app/controllers/ResponseController.scala

```diff
@@ -76,10 +58,14 @@ class ResponseController @Inject()(
         .fold(
           hasErrors => renderer.render("response.njk", json(hasErrors)).map(BadRequest(_)),
           (value: ResponseModel) => {
-            println(s"************* ArrivalId ${value.arrivalId} ************")
-            println(s"************* Version ${value.version} *************")
-            println(s"************* messageType ${value.messageType} *************")
-            destinationConnector.goodsReleased(goodsReleasedXml, value.arrivalId, value.version)
+            val xmlToSend = value.messageType match {
+              case x if x == "goodsReleased" || x == "goodsRejected" => goodsReleasedXml
```

Comment:
This is going to send back the wrong message id for goods released or goods rejected message

Reply:
removed rejected